### ISSUE TITLE
No warning for validator

### ISSUE THREAD

**Issue by zannager** [validator/validator#1699]
URL being validated or code to reproduce error:

```html
<!doctype html>
<html>
<head>
  <meta charset="utf-8">
  <title>div and span elements</title>
</head>
<body>
  <div>*** DIV 1: Some content here ***</div>
  <div>*** DIV 2: Following right after div 1 ***</div>
  <span>*** SPAN 1: Following right after div 2 ***</span>
  <div>
    *** DIV 3: Following right after span 1 
    <span>*** SPAN 2: INSIDE div 3 ***</span>
    Continue content of div 3 ***
  </div>

```
paste error messages into code block
````
Warning: Consider adding a lang attribute to the html start tag to declare the language of this document.

[From line 1, column 16; to line 3, column 6](https://validator.w3.org/nu/#l3c6)

type html>↩↩<head>↩  <me

For further guidance, consult [Declaring the overall language of a page](https://www.w3.org/International/techniques/authoring-html.en?open=language&open=textprocessing#textprocessing) and [Choosing language tags](https://www.w3.org/International/techniques/authoring-html.en?open=language&open=langvalues#langvalues).

If the HTML checker has misidentified the language of this document, please [file an issue report](https://github.com/validator/validator/issues/new) or [send e-mail to report the problem](mailto:www-validator@w3.org).

```

**Comment by nschonni** [comment]
The error is telling you you're missing the `lang` on the HTML tag and points you to documentation

**Comment by zannager** [comment]
Thanks, @nschonni. 

I just started testing this to see if it can find an error, I understand the part that I'm missing the `lang `on the HTML tag but noticed that I intentionally omitted my closing body and html tag and yet it did not warn me about it, or I'm probably missing something.
